frontend/src/pages/search/index.tsx

```diff
@@ -231,25 +227,11 @@ const Search = () => {
   }, [hits]);
 
   useDidUpdateEffect(() => {
-    setOffset(searchCriteria?.offset);
-    setCurrentCategoryIndex();
+    window.scrollTo(0, 0);
+    // setOffset(searchCriteria?.offset);
```

Comment:
I don't feel confident to review this change from cold.

Reply:
Sorry didn't mean to leave this comment in. I've been trying to remove the reliance on setting component state in multiple places.